proposals/lip-0048.md

```diff
@@ -51,16 +44,14 @@ The minimum part of the fee should be burned, this is to avoid that validators c
 We define the following constants:
 
 | Name                       | Type    | Value                                                                 | Description                                                       |
-|----------------------------|---------| ----------------------------------------------------------------------|-------------------------------------------------------------------|
+|----------------------------|---------| -------------------------------------------------------------------|-----------------------------------------------------------|
 | **General Constants**      |         |                                                                       |                                                                   |
-| `MODULE_ID_FEE`            | uint32  | 1                                                                     | Module ID of the Fee module.                                      |
-| **Configurable Constants** |         |                                                                       |                                                                   |
+| `MODULE_ID_FEE`            | bytes  | `0x00000001`                                                                     | Module ID of the Fee module.                                      |
```

Comment:
should define module name `fee` rather than ID

Reply:
Fixed.